app/views/templates/_history.html.erb

```diff
@@ -0,0 +1,25 @@
+<% unless history.empty? %>
+  <% history.each do |audit| %>
+    <% next unless audit_template? audit %>
+    <div class='row'>
+      <div class='col-md-6'>
+        <strong><%= audit_user(audit) %></strong>
+        <p>
+          <% unless audit.comment.blank? %>
+            <%= audit.comment %>
+          <% else %>
+            <em><%= _('No comment provided') %></em>
+          <% end %>
+        </p>
+      </div>
+      <div class='col-md-2 ra'><h6><%= audit_time audit %></h6></div>
+      <div class='col-md-8 audit-content ra'>
+        <%= link_to_function icon_text("retweet", _("Revert")), "tfm.editor.revertTemplate(this)", :class => 'btn btn-default', :data => {:url => revision_provisioning_templates_url, :version => audit.id } %>
```

Comment:
Maybe the restart icon? ![image](https://user-images.githubusercontent.com/7757/29552572-3864e31a-8718-11e7-93fe-80c6bce38735.png)